### DISCUSSION TITLE
Thank you very much!

### DISCUSSION THREAD

**Discussion by hwebergithub** [mrrfv/open-android-backup#48] [General]
Well done. Works fine one Nokia 6.1 Android 10
hweber

**Comment by technicaladvantage** [comment]
seconding a thank you.

MTP is a PITA to get copying just my files on any OS. 

This was a lifesaver for non-root <3

**Comment by NLBen** [comment]
Same here, thanks for this. It helped for a phone which due to faulty programming wrongly decided to call a 'no-connection' on the wi-fi while connection was in fact established. Due to the faulty error I couldn't use the regular back-up methods anymore.
And I really wasn't planning to go root, so not needing to root was a lifesaver.

**Comment by cnfcomps** [comment]
Thanks very much for providing this great backup tool. I was able to backup a Sony Xperia XA1 (G3121) with it.
